ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_logic.py

```diff
@@ -642,13 +642,13 @@ def test_jax_numpy_isinf(
     test_with_out=st.just(False),
 )
 def test_jax_numpy_isclose(
-    *,
-    dtype_and_input,
-    equal_nan,
-    on_device,
-    fn_tree,
-    frontend,
-    test_flags,
+        *,
+        dtype_and_input,
+        equal_nan,
+        on_device,
+        fn_tree,
+        frontend,
+        test_flags,
```

Comment:
Similarly for all the function changes above

Reply:
I updated the test_jax_numpy_logic.py to the newest version and found isfinite from line 607 to 631 was recently added to master so I kept it. And other functions are the same as the newest version. Thank you! :) Please let me know immediately if any other changes are needed!